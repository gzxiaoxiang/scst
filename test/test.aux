\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}this is a test program for org mode}{1}{section.1}}
\newlabel{sec-1}{{1}{1}{this is a test program for org mode\relax }{section.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}we can see the result when run org}{1}{subsection.1.1}}
\newlabel{sec-1-1}{{1.1}{1}{we can see the result when run org\relax }{subsection.1.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}test}{1}{subsection.1.2}}
\newlabel{sec-1-2}{{1.2}{1}{test\relax }{subsection.1.2}{}}
