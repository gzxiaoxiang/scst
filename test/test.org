I can't find the format file `pdflatex.fmt'!
fmtutil: config file `fmtutil.cnf' not foun* this is a test program for org mode
** we can see the result when run org 
**  test 
