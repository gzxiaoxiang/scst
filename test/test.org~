* this is a test program for org mode
** we can see the result when run org 
**  test 
