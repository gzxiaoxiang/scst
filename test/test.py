import os
print 'hello world!'
print "this is a test program!"
