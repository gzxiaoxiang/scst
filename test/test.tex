% Created 2016-07-01 Fri 22:35
\documentclass[11pt]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{fixltx2e}
\usepackage{graphicx}
\usepackage{longtable}
\usepackage{float}
\usepackage{wrapfig}
\usepackage{soul}
\usepackage{textcomp}
\usepackage{marvosym}
\usepackage{wasysym}
\usepackage{latexsym}
\usepackage{amssymb}
\usepackage{hyperref}
\tolerance=1000
\providecommand{\alert}[1]{\textbf{#1}}

\title{I can't find the format file `pdflatex.fmt'!}
\author{System Administrator}
\date{\today}

\begin{document}

\maketitle

\setcounter{tocdepth}{3}
\tableofcontents
\vspace*{1cm}
\section{this is a test program for org mode}
\label{sec-1}
\subsection{we can see the result when run org}
\label{sec-1-1}
\subsection{test}
\label{sec-1-2}

\end{document}
