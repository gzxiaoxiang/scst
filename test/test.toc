\contentsline {section}{\numberline {1}this is a test program for org mode}{1}{section.1}
\contentsline {subsection}{\numberline {1.1}we can see the result when run org}{1}{subsection.1.1}
\contentsline {subsection}{\numberline {1.2}test}{1}{subsection.1.2}
