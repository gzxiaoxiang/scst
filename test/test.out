\BOOKMARK [1][-]{section.1}{this is a test program for org mode}{}
\BOOKMARK [2][-]{subsection.1.1}{we can see the result when run org}{section.1}
\BOOKMARK [2][-]{subsection.1.2}{test}{section.1}
